src/components/VersionCheck.vue

```diff
@@ -136,7 +136,7 @@ export default {
     },
 
     incompatibleDriver() {
-      return driverPreVOD() && firmwareSupportsMix2();
+      return driverPreVOD() && firmwareSupportsMix2() && isWindowsWithDevice();
```

Comment:
Probably worth changing `firmwareSupportsMix2` to return false if there's no active device, rather than needing extra checks for it (that's my bad, i should have set that to false rather than true when adding it)